api/v1/serializers/players-serializer.js

```diff
@@ -0,0 +1,54 @@
+const appRoot = require('app-root-path');
+const decamelize = require('decamelize');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { apiBaseUrl, resourcePathLink } = appRoot.require('utils/uri-builder');
+
+const playerResourceProp = openapi.definitions.PlayerResource.properties;
+const playerResourceType = playerResourceProp.type.enum[0];
+const playerResourceKeys = _.keys(playerResourceProp.attributes.properties);
+const playerResourcePath = 'players';
```

Comment:
`playerResourcePath` should be the path to the `players` resource. Since that path is `/games/{gameId}/players`, setting the value to `players` could be misleading, since it would seem like the resource path is `/players`. Could you handle this? One option could be to make this variable a function or to just rename the variable